be/src/exec/tablet_sink.cpp

```diff
@@ -1287,6 +1299,61 @@ Status OlapTableSink::try_close(RuntimeState* state) {
     }
 }
 
+Status OlapTableSink::_fill_auto_increment_id(vectorized::Chunk* chunk) {
+    _auto_increment_slot_id = -1;
+    _has_auto_increment = false;
+
+    // no auto increment column.
+    if (_auto_increment_column_name.size() == 0) {
+        return Status::OK();
+    }
+
+    for (auto &slot : _output_tuple_desc->slots()) {
+        if (slot->col_name() == _auto_increment_column_name) {
+            _fill_auto_increment_id_internal(chunk, slot, _schema->table_id());
+            break;
+        }
+    }
```

Comment:
this for loop is inefficient, should optimize

Reply:
have optimized out the string comparison